### PR TITLE
Mhd story 12 past orders

### PR DESCRIPTION
**Pull request by mdevoe12**
Added table for orders with foreign key to users
Migrated
Feature tests for authenticated user having orders
Authenticated user does not see another user's orders
Model tests for orders and users updated for relationship validation

@katiekeel @emcooper @JoelLindow

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -39,6 +39,13 @@
     t.datetime "updated_at", null: false
   end
 
+  create_table "orders", force: :cascade do |t|
+    t.bigint "user_id"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["user_id"], name: "index_orders_on_user_id"
+  end
```
Comment (emcooper) [inline db/schema.rb]: as discussed we'll need to add status and possibly other attributes

Comment (mdevoe12) [inline db/schema.rb]: 👍 

Sounds good. Should we focus on that in the next iteration @emcooper ?

Comment (JoelLindow) [inline db/schema.rb]: Brilliant implementation. Let's move on when everyone has the time to get back together on this.

I love you all.

Comment (emcooper) [inline db/schema.rb]: Great idea, Matt 🥇

**Pull request merged**